client/components/container/EditModal.jsx

```diff
@@ -112,11 +115,15 @@ class EditModal extends Component {
               <div className="modal-body">
                 <div className="panel-body">
                   <form
-                    action="#"
-                    method="post"
+                    action=""
```

Comment:
Weldone Clinton, you followed best practices in your codebase. However, try to remove unnecessary code from your codebase. Having action and method that is not used is not needed. It would be best to remove it.

Reply:
Thanks @Rhotimee for the feedback. I would take out unnecssary code in my code.